batan-client/store/games.js

```diff
@@ -1,21 +1,32 @@
 var game_mock = {
-  "game 1": {"id": "longUID", "board": {}},
-  "game 2": {"id": "longUID2", "board": {}}
+  'game 1': { id: 'longUID', board: {} },
+  'game 2': { id: 'longUID2', board: {} },
 }
 
 export const state = () => ({
   games: game_mock,
-  activeGame: "game 1"
+  activeGame: 'game 1',
 })
 
 export const mutations = {
   changeGame(state, new_active_game) {
-    if (Object.keys(state.games).includes(new_active_game)){
-      state.activeGame = new_active_game;
+    if (Object.keys(state.games).includes(new_active_game)) {
+      state.activeGame = new_active_game
     }
   },
   vuex_test(state, test_content) {
-    console.log('vuex test received');
-    console.log(test_content);
-  }
+    console.log('vuex test received')
+    console.log(test_content)
+  },
+  reviver(key, value) {
```

Comment:
If this is just a helper function and not a mutation, then I would put this as either a lambda function or a proper function